src/Form/AdminSettingsForm.php

```diff
@@ -317,6 +318,31 @@ private function buildThumbnailUrlForm() {
   }
 
   /**
+   * Display advanced configurations forms.
+   *
+   * @return array
+   *   The render array for the advanced configuration form.
+   */
+  private function buildAdvancedConfigurationForm() {
+    $settings = $this->config('acquia_lift.settings');
+
+    $form = [
+      '#title' => t('Advanced configuration'),
+      '#type' => 'details',
+      '#tree' => TRUE,
+      '#open' => FALSE,
+    ];
+    $form['content_replacement_mode'] = [
+      '#type' => 'radios',
+      '#title' => t('Content replacement mode'),
+      '#default_value' => $settings->get('content_replacement_mode'),
+      '#options' => ['trusted' => t('Trusted'), 'untrusted' => t('Untrusted')],
+    ];
+
+    return $form;
+  }
```

Comment:
FYI:  Added a sub-task regarding adding descriptions to the module: https://backlog.acquia.com/browse/LEX-1227